src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -1,169 +1,115 @@
-import React, { useCallback, useEffect, useMemo, useState } from "react";
+import React, { useCallback } from "react";
+import classNames from "classnames";
+
+import { User } from "types/User";
 
-import { MessageToTheBandReaction, Reactions } from "utils/reactions";
 import { WithId } from "utils/id";
 
-import { useSelector } from "hooks/useSelector";
 import { useProfileModalControls } from "hooks/useProfileModalControls";
-import { useReactions } from "hooks/useReactions";
-import { useVenueId } from "hooks/useVenueId";
 
-import {
-  DEFAULT_PARTY_NAME,
-  DEFAULT_PROFILE_IMAGE,
-  RANDOM_AVATARS,
-} from "settings";
+import { UserReactions } from "components/molecules/UserReactions";
 
-import { User } from "types/User";
+import { UserAvatar } from "components/atoms/UserAvatar";
 
-// @debt remove styled-components in favour of using our standard scss patterns
-import * as S from "./UserProfilePicture.styles";
 import "./UserProfilePicture.scss";
 
+// const generateRandomAvatarUrl = (id: string) =>
+//   "/avatars/" +
+//   RANDOM_AVATARS[Math.floor(id?.charCodeAt(0) % RANDOM_AVATARS.length)];
+
+// export interface AvatarUrlProps {
+//   user?: WithId<User>;
+//   miniAvatars?: boolean;
+// }
+
+// const getAvatarUrl = ({ user, miniAvatars }: AvatarUrlProps): string => {
+//   const { id: userId, anonMode, pictureUrl } = user ?? {};
+//
+//   if (!userId || anonMode) {
+//     return DEFAULT_PROFILE_IMAGE;
+//   }
+//
+//   if (!miniAvatars && pictureUrl) {
+//     return pictureUrl;
+//   }
+//
+//   // @debt how is this intended to be used? Why doesn't it use the profile image? It seems to be something set on venue config..
+//   if (miniAvatars) {
+//     return generateRandomAvatarUrl(userId);
+//   }
+//
+//   return DEFAULT_PROFILE_IMAGE;
+// };
+
```

Comment:
Is this going to be deleted? I think it needs some kind of comment

Reply:
Unsure if deleted or debt comment yet. It seemed to work well at first, then stopped working well later. Not sure what happened, but it seemed to end up having to reload the image whenever a user moved/etc. At the very least will add the comment.

Reply:
Added `@debt` comments to explain the commented code blocks in https://github.com/sparkletown/sparkle/pull/1323/commits/5f2acbd92b4f96c175bf39ab1ffc0c0bb7d92624